test/e2e/suite_test.go

```diff
@@ -40,7 +40,7 @@ import (
 )
 
 const (
-	timeout  = 1 * time.Minute
+	timeout  = 30 * time.Second
```

Comment:
why? are we not concerned this will make it flaky?

Reply:
Timeout is shared by integration-test and e2e-test, 1 minute is too long for waiting integration test to fail if we have. We can fasten this if necessary, but right now, 30s is enough for what I'm testing. (I added some optimizations like update the name rather than the image which will slow down the rolling update process)